.azure-pipelines.yml

```diff
@@ -0,0 +1,57 @@
+variables:
+  CONDASOURCE: "~/miniconda3/etc/profile.d/conda.sh"
+
+jobs:
+- job: 'Testing'
+  displayName: 'Unit Testing With Programs'
+  pool:
+    name: qcengine
+  timeoutInMinutes: 360
+  strategy:
+    matrix:
+      Python36:
+        python.version: '3.6'
+      Python37:
+        python.version: '3.7'
+    maxParallel: 3
+
+  steps:
+    - script: |
+        echo '##vso[task.setvariable variable=ENVPREFIX]$(Agent.TempDirectory)/testnv$(python.version)'
+      displayName: "Worker Variable Assignment"
+
+      # Additional info about the build
+    - script: |
+        uname -a
+        free -m
+        df -h
+        ulimit -a
+      displayName: "Build Info"
+
+    # Setup python environment
+    - script: |
+        source $(CONDASOURCE)
+        # Create test environment for package
+        conda env create -p $ENVPREFIX python=$(python.version) -f devtools/conda-envs/psi.yaml
```

Comment:
Ah great, so not pure Bash? I guess that has some upsides and downsides.

Reply:
Its not pure bash, its able to interface with the local environment like bash, but I can also interact with the Azure Worker variables which are all parsed on script creation I think. Might not be the case, but I can access both of them all the same.